_posts/2024-12-04-enable-pluggable-storage-in-opensearch-vectordb.md

```diff
@@ -0,0 +1,255 @@
+---
+layout: post
+title:  "Deep dive: Enabling pluggable storage in the OpenSearch vector engine"
+authors:
+ - kdooyong
+ - navneev
+ - vamshin
+ - kolchfa
+date: 2024-12-16
+categories:
+ - technical-posts
+ - community
+meta_keywords: OpenSearch strategic enhancements, OpenSearch core engine, vector search engine
+meta_description: Showcase how a pluggable storage was enabled in OpenSearch vector db with a practical example of searchable snapshot which was not working in the past.
```

Comment:
Please update the meta with the following: 

_meta_keywords:_ OpenSearch vector engine, vector search, pluggable storage, Faiss, NMSLIB, k-NN search, searchable snapshots, performance optimization

_meta_description:_ Discover how OpenSearch enhanced its vector engine with a new abstraction layer, improving storage flexibility and enabling searchable snapshots for vector indexes without compromising performance.

Reply:
Hi @pajuric 
Updated accordingly. Thank you.